events/janus_wsevh.c

```diff
@@ -839,3 +840,55 @@ static int janus_wsevh_callback(struct lws *wsi, enum lws_callback_reasons reaso
 	}
 	return 0;
 }
+
+/**
+ * Implements the connecting attempt to the upstream websocket server
+ * sets the connection result (lws_client_connect_info) to static wsi
+ */
+static void janus_wsevh_connect_attempt(lws_sorted_usec_list_t* sul) {
```

Comment:
Same here (`*` next to the pointer, not type).

Reply:
ack